### ISSUE TITLE
Area Line Chart render animation is jerky

### ISSUE THREAD

**Issue by coderBeat** [Abhinandan-Kushwaha/react-native-gifted-charts#845]
### Description

Rendering the Area line chart in **Android** for the first time, runs the animation in very jerky way, not smooth.
But after rendering is finished, it is smooth.

### Steps to reproduce

1. Add isAnimated as prop
2. Render the line chart

### version of react-native-gifted-charts

1.4.39

### React Native version

0.75.2

### Platforms

Android

### Workflow

React Native

**Comment by Abhinandan-Kushwaha** [comment]
Hi @coderBeat Can you share the code? Did you test on Emulator or real device?

**Comment by coderBeat** [comment]
I tested on both Emulator and the android device:

```
<LineChart
              isAnimated
              areaChart
              curved
              rotateLabel
              disableScroll
              formatYLabel={formatYLabel}
              data={happinessData}
              height={120}
              maxValue={100}
              rulesConfigArray={rulesConfigArrayLine}
              stepValue={12.5}
              initialSpacing={20}
              endSpacing={10}
              thickness={5}
              xAxisLabelsVerticalShift={20}
              lineSegments={lineSegments}
              areaGradientId="ag"
              areaGradientComponent={lineGradient}
              pointerConfig={pointerConfigs}
              curveType={0}
            />
```

**Comment by CavalcanteLeo** [comment]
I think the same is happening to me
https://github.com/Abhinandan-Kushwaha/react-native-gifted-charts/issues/849

**Comment by Abhinandan-Kushwaha** [comment]
Hi @coderBeat What is the size of your data array?

**Comment by coderBeat** [comment]
5 elements only

`[{"customDataPoint": [Function customDataPoint], "label": "31.05.21", "pointerShiftX": 0, "value": 46}, {"customDataPoint": [Function customDataPoint], "label": "30.06.21", "pointerShiftX": 0, "value": 44}, {"customDataPoint": [Function customDataPoint], "label": "31.07.24", "pointerShiftX": 0, "value": 51}, {"customDataPoint": [Function customDataPoint], "label": "31.08.21", "pointerShiftX": -50, "value": 56}, {"customDataPoint": [Function customDataPoint], "label": "Current", "pointerShiftX": -120, "value": 48}]`

**Comment by Abhinandan-Kushwaha** [comment]
Hi @coderBeat 
Thanks for reporting this issue. It has been fixed from versions `1.4.42` onwards. 🎉

Please use the latest version of the library.

**Comment by coderBeat** [comment]
I have tried the latest version, but still the initial rendering **animations** are not smooth.
